### DISCUSSION TITLE
Is there a way to disable ligatures?

### DISCUSSION THREAD

**Discussion by Looky1173** [lapce/lapce#1347] [Q&A] [answered]
I've combed through Lapce's settings, but I couldn't find a way to disable ligatures. Is it currently possible to do this? [This article](https://practicaltypography.com/ligatures-in-programming-fonts-hell-no.html) explains in detail why some people would rather disable code ligatures.

**Comment by panekj** [comment, marked as answer]
Currently it's not possible
